controller/server/main.go

```diff
@@ -138,6 +144,66 @@ func validatePath(path string) (string, error) {
 	return path, nil
 }
 
+func (s *server) CreateTopology(ctx context.Context, req *cpb.CreateTopologyRequest) (*cpb.CreateTopologyResponse, error) {
+	log.Infof("Received CreateTopology request: %v", req)
+	bp, err := validatePath(req.Topology.Name)
+	if err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, "invalid topology path: %v", err)
+	}
+	p := topo.TopologyParams{
+		TopoName:       req.Topology.Name,
+		TopoNewOptions: []topo.Option{topo.WithBasePath(filepath.Dir(bp))},
+		Kubecfg:        kubecfg,
```

Comment:
This value should be the kubecfg provided: https://github.com/google/kne/blob/3388d2b60b540d51ad693acff6b5f81fedadd0f6/proto/controller.proto#L126

or the defaultKubeCfg if empty string